sovrin_client/test/cli/test_suspension.py

```diff
@@ -31,7 +32,7 @@ def tbgAdded(be, do, trusteeCli, nymAddedOut):
     vals = copy(vals)
     vals['target'] = vals['newTGBIdr']
     be(trusteeCli)
-    do('send NYM dest={newTGBIdr} role=TGB',
+    do('send NYM dest={{newTGBIdr}} role={role}'.format(role=Roles.TGB.name),
```

Comment:
Discussed with Lovesh:
it was discussed with Jason sometime, and approved by him. An example why we need it: sending POOL_UPGRADE txn.